content/get-started/using-git/splitting-a-subfolder-out-into-a-new-repository.md

```diff
@@ -14,6 +14,13 @@ versions:
   ghec: '*'
 shortTitle: Splitting a subfolder
 ---
+ 
+ {% tip %}
+
+   **Note:** Make sure your git >= 2.22.0 otherwise `git filter-repo` will not work.
+
+ {% endtip %}
```

Comment:
```suggestion
{% note %}

 **Note:** You need Git version 2.22.0 or later to follow these instructions, otherwise `git filter-repo` will not work.

{% endnote %}
```